docs/can-validate.md

```diff
@@ -0,0 +1,25 @@
+@module {Object} can-validate
+@parent can-ecosystem
+@group can-validate/methods Methods
+@group can-validate/types Types
+@package ../package.json
+
+Shared utilities and type definitions to process validation errors.
+
+@option {function} formatErrors Formats errors.
```

Comment:
This should be documented independently, not in here.  Plus there should be a `@type` describing what this module exports, which is an object.